File: src/app/model/round.ts
import {Phase} from './phase';
import {Card} from './card';
import {Player} from './player';

export class Round {
  id: number;
  cards: Card[];
  playersInRound: Player[];
  currentPhase: Phase;
  button: number;
  isFinished: boolean;
  pot: number;
}

File: app/src/main/java/be/kdg/mobile_client/model/Round.java
package be.kdg.mobile_client.model;

import java.util.List;

import lombok.Data;

@Data
public class Round {
    private int id;
    private List<Card> cards;
    private List<Player> playersInRound;
    private Phase currentPhase;
    private int button;
    private boolean isFinished;
    private int pot;
}
